CMakeLists.txt

```diff
@@ -101,7 +101,9 @@ else()
     find_path(Protobuf_INCLUDE_DIR google/protobuf/service.h
               PATHS ${_PROTOBUF_INSTALL_PREFIX}/include
               NO_DEFAULT_PATH)
-    find_package(Protobuf REQUIRED)
+    # Protobuf 2.5.0 is no more compatible with onnx, and 2.7.0 not yet released.
+    # https://github.com/protocolbuffers/protobuf/releases
+    find_package(Protobuf 2.6.1 REQUIRED)
```

Comment:
The version parameter in `find_package` is tricky, it's defined package by package. I am not sure if it is a good idea to use it here. cc: @zrphercule @bddppq @snnn

Reply:
If you built protobuf from source then 2.6.1 is good. But when it come to packages the version is all over the place. Some Conda packages on Windows are broken, so we recently updated instructions to use 3.9.2 because it works. I think updating instructions is a better way of addressing it.